Makefile

```diff
@@ -32,6 +32,21 @@ build-linux:
 	go build -v --ldflags="-w -X main.Version=$(VERSION) -X main.Revision=$(REVISION)" \
 		-o bin/linux/amd64/chartmuseum cmd/chartmuseum/main.go  # linux
 
+build-armv7: export GOOS=linux
+build-armv7: export GOARCH=arm
+build-armv7: export GOARM=7
+build-armv7: export CGO_ENABLED=0
+build-armv7: export GO111MODULE=on
+build-armv7: export GOPROXY=$(MOD_PROXY_URL)
+build-armv7:
+	go build -v --ldflags="-w -X main.Version=$(VERSION) -X main.Revision=$(REVISION)" \
+		-o bin/linux/armv7/chartmuseum cmd/chartmuseum/main.go  # linux
+
+container-armv7: build-armv7
+container-armv7:
+	docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
@cryptk do we need this line?

Reply:
Yes, this line is needed to configure docker itself to be able to run non-native binaries through qemu for the purposes of the build.  Without this line, you would not be able to execute armhf (or other platform) binaries, and the docker build would fail (unless you happened to actually be running the build on an armv7 platform)